scripts/generator/graphql_generator/csharp/Editor/ShopifyOnboardingPanel.cs.erb

```diff
@@ -0,0 +1,114 @@
+﻿using System.Net;
+
+namespace <%= namespace %>.SDK {
+    using System.Linq;
+    using UnityEditor;
+    using UnityEngine;
+
+    public class ShopifyOnboardingPanel : EditorWindow {
+        private const string HasSeenShopifyBuyPopupEditorPrefsKey = "has_seen_shopify_buy_popup";
+
+        [InitializeOnLoadMethod]
+        public static void OnLoad() {
+            if (PopupHasBeenShown()) return;
+            ShowWindow();
+        }
+
+        [MenuItem("Shopify/Help")]
+        public static void ShowWindow() {
+            var window = EditorWindow.GetWindow<ShopifyOnboardingPanel>();
+            window.ShowPopup();
+
+            EditorPrefs.SetBool(HasSeenShopifyBuyPopupEditorPrefsKey, true);
+        }
+
+        private static bool PopupHasBeenShown() {
+            return EditorPrefs.GetBool(HasSeenShopifyBuyPopupEditorPrefsKey, false);
+        }
+
+        private const string SettingsFileName = "Shopify Buy Visual GUI Settings";
+
+        private ShopifyEditorStyleSettings _settings;
+
+        private ShopifyEditorStyleSettings Settings {
+            get {
+                if (_settings != null) return _settings;
+                
+                Debug.Log(typeof(ShopifyEditorStyleSettings).FullName);
+                var assetGuids = AssetDatabase.FindAssets(SettingsFileName);
```

Comment:
I load this guy by file name, it could in theory break if someone created an asset called "Shopify Buy Visual GUI Settings" but I think its unlikely???